example/CMakeLists.txt

```diff
@@ -2,6 +2,19 @@ zxing_add_package_stb()
 
 include (GNUInstallDirs)
 
+add_definitions(
```

Comment:
Please use `target_compile_definitions` for the qt-targets only. Also what is the motivation to define all of those? You mentioned `-DQT_NO_SIGNALS_SLOTS_KEYWORDS` and `-DQT_NO_KEYWORDS` but why all the others?

Reply:
The others are to ensure QString is efficiently created